azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/CertificatePatchResource.java

```diff
@@ -376,4 +382,24 @@ public CertificatePatchResource withServerFarmId(String serverFarmId) {
         return this;
     }
 
+    /**
+     * Get cNAME of the certificate to be issued via free certificate.
```

Comment:
cName or CNAME? be consistent

Reply:
Should be CNAME.

Seems autorest.java issue. Swagger is good.